json-rpc.md

```diff
@@ -1065,8 +1065,8 @@ Returns information about a block by hash.
 
 ```js
 params: [
-   '0xe670ec64341771606e55d6b4ca35a1a6b75ee3d5145a99d05921026d1527331',
-   true
+   '0xdc0818cf78f21a8e70579cb46a43643f78291264dda342ae31049421c82d21ae',
+   false
```

Comment:
@acolytec3 Why change these? Is that hash correct? And I'm not sure if the `true` was better?